packages/runner-ct/src/app/BurgerIcon.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+export const BurgerIcon: React.FC = () => {
```

Comment:
Spread props to svg will be nice

Reply:
@elevatebart maybe, he added this icon. Probably a good idea